examples/web-wagmi/src/App.tsx

```diff
@@ -57,7 +57,7 @@ import { UseProfileFollowRevenue } from './revenue/UseProfileFollowRevenue';
 import { UseProfilePublicationRevenue } from './revenue/UseProfilePublicationRevenue';
 import { UsePublicationRevenue } from './revenue/UsePublicationRevenue';
 
-const { provider, webSocketProvider } = configureChains([polygon, optimism], [publicProvider()]);
+const { provider, webSocketProvider } = configureChains([polygon], [publicProvider()]);
```

Comment:
`optimism` is a leftover

Reply:
shall we use `polygonMumbai` then?

Joaquin had similar finding in his PR https://github.com/lens-protocol/lens-sdk/pull/120/files

Reply:
Good point, merging this one than and we will also keep the Joaquin changes in the #120